backend/migrations/cmd/backfill-hubspot/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	hubspotApi "github.com/highlight-run/highlight/backend/hubspot"
+	"github.com/highlight-run/highlight/backend/model"
+	public "github.com/highlight-run/highlight/backend/public-graph/graph"
+	"github.com/leonelquinteros/hubspot"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"os"
+	"sync"
+)
+
+const BatchSize = 64
+
+func main() {
+	ctx := context.Background()
+	log.WithContext(ctx).Info("setting up db")
+	db, err := model.SetupDB(ctx, os.Getenv("PSQL_DB"))
+	if err != nil {
+		log.WithContext(ctx).Fatalf("error setting up db: %+v", err)
+	}
+
+	r := public.Resolver{
+		DB:         db,
+		HubspotApi: hubspotApi.NewHubspotAPI(hubspot.NewClient(hubspot.NewClientConfig())),
+	}
+
+	var admins []model.Admin
+
+	wg := sync.WaitGroup{}
+	inner := func(tx *gorm.DB, batch int) error {
+		wg.Add(1)
+		func() {
+			for _, admin := range admins {
+				if admin.Email == nil || admin.UserDefinedRole == nil || admin.UserDefinedPersona == nil || admin.FirstName == nil || admin.LastName == nil || admin.Phone == nil || admin.Referral == nil {
+					log.WithContext(ctx).WithField("AdminID", admin.ID).Error("admin is not filled out")
+					continue
+				}
+				hubspotContactId, err := r.HubspotApi.CreateContactForAdmin(
```

Comment:
Is this hubspot call (and the rest of the hubspot calls) part of some sign up flow? Can we call some existing function that does this work? Mostly thinking that keeping the hubspot logic for new users in sync would be difficult if it's duplicate.

Reply:
good idea, but it's part of a more complicated workflow using about you form data. this seems pretty specific to the backfilling logic so i'm going to keep this separate.

<img width="765" alt="Screenshot 2023-03-15 at 1 29 15 PM" src="https://user-images.githubusercontent.com/1351531/225434626-baaf4a5b-6f46-4f8b-bac2-5b9b2bfb2b20.png">